symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkBotConfig.java

```diff
@@ -2,22 +2,11 @@
 
 import lombok.Getter;
 import lombok.Setter;
-import org.apache.commons.lang3.StringUtils;
 
 @Getter
 @Setter
-public class BdkBotConfig {
+public class BdkBotConfig extends BdkAuthenticationConfig {
 
     private String username;
-    private String privateKeyPath;
-    private String certificatePath;
```

Comment:
in term of configuration should'nt we have something like:

username: "mybot"
certificate:
  path:
  password

repeating the same prefix is often the sign that a sub structure should be used

but then to be consistent you would wonder if privateKeyPath should a privateKey item with a path sub item....
could we have other settings for a private key?